src/script.rs

```diff
@@ -77,11 +77,10 @@ impl Script {
     #[inline]
     pub fn invoke<T: FromRedisValue>(&self, con: &ConnectionLike) -> RedisResult<T> {
         ScriptInvocation {
-                script: self,
-                args: vec![],
-                keys: vec![],
-            }
-            .invoke(con)
+            script: self,
```

Comment:
Sorry for the churn caused by accidentally having rustfmt on. I could revert some of the changes If you want but I fear the files I modified may have to stick. (Would be nice to run through rustfmt on the whole repo though in my opinion).

Reply:
@Marwes i'm fine enforcing rustfmt. I tried to do it in the past but it completely broke with some of the macros at the point. Need to reinvestigate this.